cpp/roaring64map.hh

```diff
@@ -1722,8 +1720,8 @@ class Roaring64MapSetBitBiDirectionalIterator {
     typedef int64_t difference_type;
     typedef Roaring64MapSetBitBiDirectionalIterator type_of_iterator;
 
-    Roaring64MapSetBitBiDirectionalIterator(const Roaring64Map &parent,
-                                            bool exhausted = false)
+    explicit Roaring64MapSetBitBiDirectionalIterator(const Roaring64Map &parent,
```

Comment:
I don't mind the explicit but it is potentially a breaking change. Can you justify the explicit?

We would rather not break people's code.

Reply:
Sure. This is added since the corresponding constructor in `Roaring` comes with an _explicit_, hence an _explicit_ here could give us more symmetry.
On the other hand, you are right, backward compatibility matters.
So I'm now neutral about this change and waiting for more points of view.

Reply:
@cubicYYY We can't assume that CRoaring is only used by projects on GitHub. So if we check and find something then we know that it is in use, but if we don't find anything, we don't know whether it is in use.

Reply:
@lemire That makes sense. If backward compatibility is a must, what about adding a brief comment instead of actually making it `explicit`?

Reply:
Honestly, I recommend just dropping the issue and removing the explicit. The motivation seems weak:

> This is added since the corresponding constructor in Roaring comes with an explicit, hence an explicit here could give us more symmetry.

Can we come up with one example where this change helps anyone?

If a change is not helpful, and if it is purely for aesthetic, then I recommend dropping it. Even if there is a very tiny chance that it breaks someone's code, it is worse than an aesthetic issue nobody is ever going to notice in the real world.

In other words, it seems to me that this is not worth changing.

Reply:
Roger!